### DISCUSSION TITLE
Install fail! Error: [prisma@5.14.0 › @prisma/engines@5.14.0] Can't find package @prisma/engines's version: 5.14.0

### DISCUSSION THREAD

**Discussion by lik778** [prisma/prisma#24202] [Q&A]
### Bug description

 > [6/9] RUN cnpm install:                                                                                                                                                                  
9.786 ✖ Install fail! Error: [prisma@5.14.0 › @prisma/engines@5.14.0] Can't find package @prisma/engines's version: 5.14.0                                                                  
9.787 Error: [prisma@5.14.0 › @prisma/engines@5.14.0] Can't find package @prisma/engines's version: 5.14.0                                                                                  
9.787     at resolve (/usr/local/lib/node_modules/cnpm/node_modules/npminstall/lib/download/npm.js:118:11)                                                                                  
9.787     at async module.exports (/usr/local/lib/node_modules/cnpm/node_modules/npminstall/lib/download/npm.js:22:19)                                                                      
9.787     at async module.exports (/usr/local/lib/node_modules/cnpm/node_modules/npminstall/lib/download/index.js:19:10)
9.787     at async _install (/usr/local/lib/node_modules/cnpm/node_modules/npminstall/lib/install_package.js:122:16)
9.787     at async install (/usr/local/lib/node_modules/cnpm/node_modules/npminstall/lib/install_package.js:23:12)
9.787     at async mapper (/usr/local/lib/node_modules/cnpm/node_modules/npminstall/lib/install_package.js:270:9)
9.787 npminstall version: 7.12.0
9.787 npminstall argv: /usr/local/bin/node /usr/local/lib/node_modules/cnpm/node_modules/npminstall/bin/install.js --fix-bug-versions --china --userconfig=/root/.cnpmrc --disturl=https://cdn.npmmirror.com/binaries/node --registry=https://registry.npmmirror.com

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```
 "@prisma/client": "5.13.0",
"prisma": "^5.13.0",
```

**Comment by janpio** [comment]
As you can see on https://www.npmjs.com/package/@prisma/engines, 5.14.0 of `@prisma/engines` has properly been published. It seems the npmirror you are using is outdated. There is unfortunately nothing we can do about that.

**Reply by janpio** [reply]
On the other hand, https://npmmirror.com/package/@prisma/engines also lists 5.14.0 as the latest version - did the problem maybe already resolve itself?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
